src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -49,6 +55,9 @@ public class DriveTrain extends SubsystemBase {
      */
     private static final double WHEELBASE_METERS = Units.inchesToMeters(24.625);
 
+    // TODO confirm if this year's robot is a square base
+    private static final double DRIVE_BASE_RADIUS_METERS = WHEELBASE_METERS / Math.sqrt(2);
```

Comment:
This should be the general solution in terms of TRACKWIDTH_METERS and WHEELBASE_METERS.